plugins/webapi/value/sendtransactionbyjson/handler.go

```diff
@@ -121,6 +122,17 @@ func NewTransactionFromJSON(request Request) (*transaction.Transaction, error) {
 
 	// prepare transaction
 	tx := transaction.New(transaction.NewInputs(inputs...), transaction.NewOutputs(outputs))
+	accessMana, err := mana.IDFromPubKey(request.AccessMana)
+	if err != nil {
+		return nil, err
+	}
+	tx.SetAccessManaNodeID(accessMana)
+	consensusMana, err := mana.IDFromPubKey(request.ConsensusMana)
```

Comment:
check if `accessMana`/`consensusMana` is in the list of allowed nodeIDs, return an error otherwise.